ViewGenerator/tools/webpack/Rules/Files.ts

```diff
@@ -23,11 +23,50 @@ const getResourcesRuleSet = (assemblyName?: string, pluginsBase? : string): Rule
                         //
                         // Context represents the path of the project being built by webpack, e.g. "C:\Git\Path\to\Project\"
                         //
-                        const buildUrl = (url: string, resourceBase: string): string => {
+                        const PackagesFolder = "/packages/";
+                        const isOutSystemsPackagesWithOldNaming = (url) => {
+                            // means that the package name is (example) "web_feedback-message" or "web_image" instead of "web-feedback-message" or "web-image"
+                            const OutSystemsPackagesWithOldNaming = [
```

Comment:
noooo :(
this is a very specific knowledge that should not be here

Reply:
cant we have another approach?

Reply:
I have no clue how to do this differently 😞 

@alvesmiguel1, @mpm-os do you have a different approach?

Reply:
I raised this concern when implementing this. We need to fix the packages with the _ - ...